news/_posts/performance/2020-12-07-performance-and-prestashop.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title:  "Performance and PrestaShop"
+subtitle: "Sorry, Goose, but it's time to buzz the tower."
```

Comment:
I'm afraid I did not understand this (so I googled it) 😅  so maybe more people will not understand it?

Reply:
Well it's jut a pop culture joke, not sure it's important if (young) people don't catch it ;)

Reply:
![https://media.giphy.com/media/D16XHdsB1PBxm/giphy.gif](https://media.giphy.com/media/D16XHdsB1PBxm/giphy.gif)